client/src/js/actions/recipe.js

```diff
@@ -407,11 +418,9 @@ export const apifavoriteRecipe = id =>
       url: `/api/v1/recipe/${id}/favorite`
     });
     request.then(() => {
-      console.log('favorited');
       dispatch(favoriteRecipeSuccess(id));
     }).catch((err) => {
       if (err && err.data) {
```

Comment:
### Critical

What's the purpose of this check? I believe you want to handle error appropriately, what happens when `err.data` doesn't exist?

I believe `err` will always exist when it gets into the catch block sha.